util/src/dbutil.cpp

```diff
@@ -224,5 +281,60 @@ std::shared_ptr<odb::database> createDatabase(const std::string& connStr_)
   return db;
 }
 
+void createTables(
+  std::shared_ptr<odb::database> db_,
+  const std::string& sqlDir_)
+{
+  runSqlFiles(db_, sqlDir_,
+    [](const std::string& s_){
+      return removeByRegex(removeByRegex(removeByRegex(s_,
+        "CREATE UNIQUE INDEX", ";"),
+        "CREATE INDEX", ";"),
+        "ALTER TABLE", ";");
+    },
+    "Creating tables from file");
+}
+
+void createIndexes(
+  std::shared_ptr<odb::database> db_,
+  const std::string& sqlDir_)
+{
+  runSqlFiles(db_, sqlDir_,
+    [](const std::string& s_){
+      return removeByRegex(removeByRegex(s_,
+        "CREATE TABLE", ";"),
+        "DROP", ";");
+    },
+    "Creating indexes from file");
+}
+
+std::string updateConnectionString(
```

Comment:
I don't believe that it is a problem if a utility function is as general as it can be. Adding a new key is used when the user didn't provide a database name in the connection string.